Model/Source/DefaultOptions.php

```diff
@@ -189,11 +189,14 @@ public function getPackageType(): int
     {
         $country      = self::$order->getShippingAddress()->getCountryId();
         $isNL         = AbstractConsignment::CC_NL === $country;
-        $keyIsPresent = array_key_exists('packageType', self::$chosenOptions);
-        $packageType  = self::$chosenOptions['packageType'];
 
-        if ($isNL && $keyIsPresent) {
-            return AbstractConsignment::PACKAGE_TYPES_NAMES_IDS_MAP[$packageType];
+        if (! empty(self::$chosenOptions)) {
```

Comment:
Willen we echt `if (! empty($shizzle))` gebruiken als dat hetzelfde is als `if ($shizzle)`?